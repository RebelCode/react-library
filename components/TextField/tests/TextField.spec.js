import React from "react";
import { render, cleanup } from "@testing-library/react";
import { TextField } from "../index";

afterEach(cleanup);

describe("TextField", () => {
  const Comp = <TextField />

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("TextField")).toBeTruthy();
  });
});
