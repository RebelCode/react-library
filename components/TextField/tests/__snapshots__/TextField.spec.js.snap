// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField should take a snapshot 1`] = `
<DocumentFragment>
  <input
    aria-label="TextField"
    data-testid="textfield"
    type="text"
    value=""
  />
</DocumentFragment>
`;
